server/src/repositories/Training/ModuleItemRepository.ts

```diff
@@ -0,0 +1,99 @@
+import { knex } from "../../db";
+import { ModuleItem } from "../../schemas/trainingSchema";
+import * as ModuleItemMap from "../../mappers/training/moduleItemMapper";
+
+export async function getModuleItem(
+  moduleItemId: number | string
+): Promise<ModuleItem | null> {
+  const knexResult = await knex("ModuleItem")
+    .select(
+      "id",
+      "module",
+      "moduleItemType",
+      "text",
+      "order"
+    )
+    .where("id", moduleItemId);
+  return ModuleItemMap.singleModuleItemToDomain(knexResult);
+}
+
+export async function getModuleItemsByModule(
+  moduleId: number | string
+): Promise<ModuleItem[]> {
+  const knexResult = await knex("ModuleItem")
+    .select(
+      "id",
+      "module",
+      "moduleItemType",
+      "text",
+      "order"
```

Comment:
This may be a controversial take but I've recently taken to the idea of not including the column names and just doing `select()`. Since we're always feeding the data through a mapper, I don't think it's needed to always be explicit about which columns.

Reply:
That makes sense 🧠